client/src/pages/HomePage/HomePage.module.scss

```diff
@@ -0,0 +1,3 @@
+.heroSection {
+  position: relative;
```

Comment:
**Controversial question:** if there would be no more styles in this file, maybe better to make styled component instead?

Reply:
i agree,  was waiting till the end of the first revision to roll back this style.